WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/BtcPriceTileViewModel.cs

```diff
@@ -1,29 +1,21 @@
-using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using ReactiveUI;
-using WalletWasabi.Fluent.Helpers;
 using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.Tiles;
 
-public partial class BtcPriceTileViewModel : TileViewModel
+public class BtcPriceTileViewModel : TileViewModel
 {
-	private readonly Wallet _wallet;
-	[AutoNotify] private string _btcPrice;
-
 	public BtcPriceTileViewModel(Wallet wallet)
 	{
-		_wallet = wallet;
-		_btcPrice = "";
+		Wallet = wallet;
+
+		UsdPerBtc = this
+			.WhenAnyValue(x => x.Wallet.Synchronizer.UsdExchangeRate)
+			.ObserveOn(RxApp.MainThreadScheduler);
 	}
 
-	protected override void OnActivated(CompositeDisposable disposables)
-	{
-		base.OnActivated(disposables);
+	public IObservable<decimal> UsdPerBtc { get; }
 
-		_wallet.Synchronizer.WhenAnyValue(x => x.UsdExchangeRate)
-			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(usd => BtcPrice = usd.FormattedFiat(format: "N0") + " USD")
-			.DisposeWith(disposables);
-	}
+	private Wallet Wallet { get; }
```

Comment:
Is this necessary to store the reference?

Reply:
Yes, to make the WhenAny syntax work in order to comply with this:
https://www.reactiveui.net/docs/guidelines/framework/use-this-on-left-of-whenany

There's a good reason behind this recommendation. WhenAnyValue does the heavy lifting for us (by walking the property expression and watching the final property).

Basically, in order to make the subscription to the observable have the the same lifecycle as the VM, it's highly recommended to do it like this. Otherwise, we would have to dispose manually instead. 

This is safer and easier.

Reply:
ok, make sense.